src/librustdoc/html/render/mod.rs

```diff
@@ -312,15 +312,15 @@ impl AllTypes {
         f.write_str(
             "<h1 class=\"fqn\">\
                  <span class=\"in-band\">List of all items</span>\
-                 <span class=\"out-of-band\">\
-                     <span id=\"render-detail\">\
-                         <a id=\"toggle-all-docs\" href=\"javascript:void(0)\" \
-                            title=\"collapse all docs\">\
-                             [<span class=\"inner\">&#x2212;</span>]\
-                         </a>\
-                     </span>
-                 </span>
-             </h1>",
+             </h1>\
+             <span class=\"out-of-band\">\
+                <span id=\"render-detail\">\
+                    <a id=\"toggle-all-docs\" href=\"javascript:void(0)\" \
+                    title=\"collapse all docs\">\
```

Comment:
nit: could could indent the `title` attribute so my brain doesn't interpret it as a tag please?